<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>684</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>646</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bitwise Calc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="cell_frame">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>490</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>490</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>490</width>
      <height>150</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="BPrev">
          <property name="text">
           <string>Prev</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Current cell Index</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="EIndex">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="BNext">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>BIN</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>OCT</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>DEC</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>HEX</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="EBin">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="EOct">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="EDec">
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="EHex">
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>490</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>490</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>490</width>
      <height>10000</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>153</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Enter number : </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>Bin</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>Oct</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_3">
            <property name="text">
             <string>Dec</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_4">
            <property name="text">
             <string>Hex</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="BSet">
        <property name="text">
         <string>SET</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="BAdd">
          <property name="text">
           <string>ADD</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BSub">
          <property name="text">
           <string>SUB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BMul">
          <property name="text">
           <string>MUL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BDiv">
          <property name="text">
           <string>DIV</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>481</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="BByte_2">
         <property name="text">
          <string>UBYTE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BWord_2">
         <property name="text">
          <string>UWORD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BDWord_2">
         <property name="text">
          <string>UDWORD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BQWord_2">
         <property name="text">
          <string>UQWORD</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="BByte">
         <property name="text">
          <string>BYTE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BWord">
         <property name="text">
          <string>WORD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BDWord">
         <property name="text">
          <string>DWORD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BQWord">
         <property name="text">
          <string>QWORD</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
